### PR TITLE
Update current branch for 7.7 release

### PR DESCRIPTION
**Pull request by jrodewig**
DO NOT MERGE THIS PR UNTIL RELEASE DAY

Changes:

* Adds `&stackcurrent` variable for the current Stack docs branch
* Sets the `&stackcurrent` variable as `7.7`
* Removes the `7.6` branch from `&stacklive`

### CODE REVIEW

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by bmorelli25** [comment]
Hey James! Can we run a rebuild on this PR on Monday? I'm hoping to avoid a repeat of 7.6: https://github.com/elastic/docs/pull/1704#issuecomment-584681901.

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by bmorelli25** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

File: conf.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@ contents:
           -
             title:      Getting Started
             prefix:     en/elastic-stack-get-started
-            current:    7.6
+            current:    7.7
```
Comment (gtback) [inline conf.yaml:106]: Going forward, would it help to create a `stackcurrent` variable, like we have for `stacklive`, so we only have to change this in one place, for all books which are versioned in line with the Stack?

Comment (jrodewig) [reply conf.yaml:106]: Thanks @gtback. I think that's a great idea.

Comment (jrodewig) [reply conf.yaml]: Went ahead and added a `&stackcurrent` variable with 1ae5b6c. Also removed `7.6` from the `&stacklive` variable as 7.6 will no longer be maintained after 7.7 releases.

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
Closed in favor of #1822.

**Pull request closed without merging**